File: bot-svc/src/main/kotlin/xyz/zelda/bot/BotApplication.java
package xyz.zelda.bot;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
import org.springframework.cloud.openfeign.EnableFeignClients;

@SpringBootApplication(exclude={DataSourceAutoConfiguration.class})
@EnableFeignClients(basePackages = {"xyz.zelda.mail", "xyz.zelda.sms", "xyz.zelda.company", "xyz.zelda.account"})
public class BotApplication {
    public static void main(String[] args) {
        SpringApplication.run(BotApplication.class, args);
    }
}

File: bot-svc/src/main/kotlin/xyz/zelda/bot/BotApplication.kt
package xyz.zelda.bot

import org.springframework.boot.Banner
import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication


@SpringBootApplication
open class BotApplication

fun main(args: Array<String>) {
    runApplication<BotApplication>(*args){
        setBannerMode(Banner.Mode.CONSOLE)
    }
}
